sap/cli/gcts_utils.py

```diff
@@ -88,6 +91,24 @@ def get_activity_rc(repo, operation: RepoActivitiesQueryParams.Operation):
     return int(activities_list[0]['rc'])
 
 
+def is_cloned_activity_success(console, repo: Repository) -> bool:
+    """Check if the cloned activity is successful"""
+    clone_rc = get_activity_rc(repo, RepoActivitiesQueryParams.Operation.CLONE)
+    if clone_rc != Repository.ActivityReturnCode.CLONE_SUCCESS.value:
```

Comment:
Please have a look at this: https://github.com/jfilak/sapcli/commit/55e7e9efc269b2286eac7d8d1f25f37995700a8a